news/_posts/2020-03-bug-bounty-program.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: post
+title:  "Let the hunt begin"
+subtitle: "It's time to make PrestaShop better that it already is!"
+date:   2020-03-11 08:00:00
+authors: [ PierreRambaud, PrestaShop ]
+icon: icon-chat
+image: /assets/images/theme/meta-logo-build.png
+tags: [bug, bounty, program, maintenance]
+---
+
+Protecting the privacy of PrestaShop's users is one of our top priority. We build the software with this goal in mind.
+That's why we decided to welcome help from the outisde through a bounty program to put our security to the test!
```

Comment:
```suggestion
That's why the PrestaShop company decided to welcome help from the outisde through a bounty program to put our security to the test!
```
please, when you use "we", make it clear who is "we". It can be the company, the core devs, your team, the author of the article + some people who took a decision, or many other combination of people and organisation.